Gravity-Notes/src/App.js

```diff
@@ -1,8 +1,11 @@
 import { Route, Routes } from "react-router-dom";
 import "./App.css";
+import "../src/Default.css";
 import { Bodycontent } from "./Components/Body-content/Body-content";
 import { Footer } from "./Components/Footer/Footer";
 import { Navbar } from "./Components/Navigation/Navbar";
+import { Login } from "./Pages/AuthWidget/Login/Login";
+import { SignUp } from "./Pages/AuthWidget/SignUp/SignUp";
 import { Homepage } from "./Pages/Homepage/Homepage";
 import { Landingpage } from "./Pages/Landing-page/Landing-page";
 
```

Comment:
you can create an index file and export all components/pages there. here you can import in single line.